ws-server/src/server.rs

```diff
@@ -309,19 +315,42 @@ async fn background_task(
 	let stop_server2 = stop_server.clone();
 	let sink = MethodSink::new_with_limit(tx, max_response_body_size);
 
+	let mut submit_ping = tokio::time::interval(ping_interval);
+	let ping_submitted_tx = Arc::new(AtomicBool::new(false));
```

Comment:
Do we really have to kill the connection if a pong was not received between the provided window?

I'm a little terrified of having more boolean flags that must be set/cleared and those are easy mess up on and so on when the refactoring the code.

Reply:
I don't think we explicitly need to handle `Pong`s if we only care about keeping the connection alive.

That would also simplify the code quite a bit server-side 😄

Reply:
cool, I guess we could add another issue for that if anyone wants kill their connection once the ping is not ACK:ed by pong with the period but it could also be a footgun that the ping period is too short with a slow connection.